compiler/parser/src/lexer.rs

```diff
@@ -18,8 +18,8 @@ use unic_ucd_ident::{is_xid_continue, is_xid_start};
 
 #[derive(Clone, Copy, PartialEq, Debug, Default)]
 struct IndentationLevel {
-    tabs: usize,
-    spaces: usize,
+    tabs: u32,
+    spaces: u32,
```

Comment:
I think there should be some sort of achievement for attempting to use 4,294,967,296 tabs or spaces for indentation, resulting in a script that's over 4 GB in size.